.github/workflows/build-fb-image.yaml

```diff
@@ -1,6 +1,11 @@
 name: Building Fluent Bit image
 
 on:
+  push:
+    branches:
+      - main
+    paths:
+      - cmd/fluent-watcher/fluentbit/VERSION
```

Comment:
I can remove all except this and the RELEASE.md update from this PR.

Maybe after you incorporate all the other stuff into your PR.